pkg/scheduler/apis/config/scheme/scheme_test.go

```diff
@@ -302,7 +298,7 @@ profiles:
 						{
 							Name: "PodTopologySpread",
 							Args: &config.PodTopologySpreadArgs{
-								DefaultingType: config.ListDefaulting,
```

Comment:
in retrospect, this looks strange, defaulting to listdefaulting without constraints.

Reply:
oh, or is that how we disable the default constraints?

Reply:
Correct. And that's the default if the feature gate is disabled.